pyproject.toml

```diff
@@ -0,0 +1,7 @@
+[tool.black]
+line-length = 120
+target-version = ['py311']
+force-exclude = '''test'''
+
+[tool.coverage.run]
+omit = ["tests/*", "main.py", "version_tag_check/github_repository.py"]
```

Comment:
Wrong project with `"version_tag_check/github_repository.py"`.

Reply:
Nice catch. Fixed in commit